practice/src/css/bases/variable.css

```diff
@@ -25,4 +25,7 @@
   --primary-background-color: #fff;
   --secondary-background-color: #fafafa;
   --tertiary-background-color: #292c39;
+  --fouth-background-color: #00afd5;
```

Comment:
Naming `fourth`, or `fifth` isn't good. You can refer to Bootstrap to define variable name https://getbootstrap.com/docs/5.2/customize/css-variables/

Besides, You shouldn't use acronyms for font-weight `--fw-bold: 700;`. It should be `--font-weight-bold: 700`